src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -553,10 +560,6 @@ void FancyZones::UpdateZoneWindows() noexcept
         return TRUE;
     };
 
-    {
-        std::unique_lock writeLock(m_lock);
-        m_zoneWindowMap.clear();

```

Comment:
Map `m_zoneWindowMap` is updated in `FancyZones::AddZoneWindow` and always up to date. No need to clear it every time.